zeek_inet_ntop.c

```diff
@@ -149,7 +149,7 @@ zeek_inet_ntop6(const u_char *src, char *dst, socklen_t size)
 	 */
 	int remaining = sizeof(tmp);
 	tp = tmp;
-	for (i = 0; i < (NS_IN6ADDRSZ / NS_INT16SZ); i++) {
+	for (i = 0; i < (NS_IN6ADDRSZ / NS_INT16SZ) && remaining > 0; i++) {
```

Comment:
Is line 165 below missing a `remaining--` ? Can't comment there.
https://github.com/zeek/zeek-3rdparty/pull/23/files#diff-c9e934660269e2c4174cbae9201b0c174dda7489eda32b772aba401893e3cc0bR165